### PR TITLE
Allow to install and remove operator via scripts

### PR DESCRIPTION
**Pull request by jiripetrlik**
## Why are these changes needed?

It allows to install and uninstall Kuberay operator using scripts. It will be useful for environments where it is not possible to use Helm.

- [ ] I've made sure the tests are passing. 
- Testing Strategy
   - [ ] Unit tests
   - [ ] Manual tests
   - [ ] This PR is not tested :(

### CODE REVIEW

File: tests/compatibility-test.py [hunk 1]
```diff
@@ -83,7 +74,7 @@ def setUpClass(cls):
             CONST.RAY_IMAGE_KEY: ray_image,
```
Comment (kevin85421) [inline tests/compatibility-test.py:74]: Should we remove `image_dict` here?

File: tests/compatibility-test.py [hunk 2]
```diff
@@ -230,7 +221,7 @@ def setUpClass(cls):
...
```
Comment (kevin85421) [inline tests/compatibility-test.py:221]: Should we remove `image_dict` here?

**Review by kevin85421 — APPROVED** [review]
LGTM. Just some nits. The codebase becomes much cleaner!

**Comment by jiripetrlik** [comment]
FYI: @sutaakar

**Comment by kevin85421** [comment]
Rerun the flaky tests. I will merge this PR after they pass.

**Comment by kevin85421** [comment]
@jiripetrlik Could you fix the CI errors?
Screen Shot 2023-10-30 at 2 13 59 PM

**Comment by jiripetrlik** [comment]
@kevin85421 
It should be fixed now. I still see some CI errors, but I think it is not related to this PR.

**Pull request merged**